modules/registry-operator-configuration-resource-overview.adoc

```diff
@@ -0,0 +1,100 @@
+// Module included in the following assemblies:
+//
+// * assembly/registry
+// * openshift_images/configuring-registry-operator.adoc
+
+
+[id="registry-operator-configuration-resource-overview-{context}"]
+= Image Registry Operator configuration
+
+The image-registry resource offers the following configuration fields.
+
+[cols="3a,8a",options="header"]
+|===
+|Component |Description
+
+|*ManagementState*
+|*Managed:* the operator will update the registry as configuration resources
```

Comment:
s/operator/Operator
^ all places

Please double-check that all the future tense is necessary - present tense often is good enough.
The : should be outside the formatting.
Double-check that each entry starts with a capital letter and ends with a period.